src/devices/BrickPi3/Movement/Motor.cs

```diff
@@ -153,6 +159,10 @@ public int GetSpeed()
         public int Speed
         { get { return GetSpeed(); } set { SetSpeed(value); } }
 
+        /// <summary>
+        /// Motor port
+        /// </summary>
+        /// <value></value>
```

Comment:
Interesting. First time I see someone explicitly add the `<value></value>` tags, which are actually required to get documented in the dotnet-api-docs repo for Properties.

The `<value>` tags is used to describe the Property data type that can the user can get and/or set.

It' clearer to explain with booleans. For example:

```cs
/// <summary>Determines if the instance is enabled.</summary>
/// <value><see langword="true" /> if enabled, <see langword="false" /> if it's disabled.</value>
public bool IsEnabled { get; }
```

Reply:
My editor was adding `<value>` but I kept removing it because I've never seen anyone do that and thought this was the tooling issue.

I'm not sure if I'll address this one everywhere but will consider changing the keywords for now at least